.github/workflows/grandpa-cw-docker-image.yml

```diff
@@ -0,0 +1,74 @@
+name: "Build and publish Grandpa Docker image"
+
+on:
+  push:
+    branches:
+      - '*'
+    tags:
+      - 'v*'
+
+env:
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  
+  build-and-publish:
+
+    name: Build & push docker image
+    
+    runs-on:
+      - self-hosted
+      - x64-monster
+    
+    concurrency:
+      group: grandpa-cw-image-${{ github.ref }}
+      cancel-in-progress: true
+    
+    strategy:
+      fail-fast: true
+
+    steps:
+    - name: Clean up
+      continue-on-error: true
+      run: |
+        sudo chown -R $USER:$USER $GITHUB_WORKSPACE
+        docker system prune --force --all --volumes
+
+    - name: Install dependencies
+      run: |
+        sudo apt-get install -y make build-essential
+
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v3
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+
+    - name: Extract metadata (tags, labels) for Docker
+      id: meta
+      uses: docker/metadata-action@v5
+      with:
+        images: ${{ env.IMAGE_NAME }}
```

Comment:
this cant' be the same like main image of `composable-ibc`.

Change:

```
env:
  IMAGE_NAME: ${{ github.repository }}
```

to something like:

```
env:
  IMAGE_NAME: composablefi/grandpa-cw
```